src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -286,7 +290,7 @@ def delete_arc_agents(release_namespace, kube_config, kube_context, configuratio
 
 def helm_install_release(chart_path, subscription_id, kubernetes_distro, kubernetes_infra, resource_group_name, cluster_name,
                          location, onboarding_tenant_id, http_proxy, https_proxy, no_proxy, proxy_cert, private_key_pem,
-                         kube_config, kube_context, no_wait, values_file_provided, values_file, cloud_name, disable_auto_upgrade, enable_custom_locations, custom_locations_oid):
+                         kube_config, kube_context, no_wait, values_file_provided, values_file, cloud_name, disable_auto_upgrade, enable_custom_locations, custom_locations_oid, onboarding_timeout="5"):
```

Comment:
Now its seconds, so 300 right?

Reply:
its updated in the latest commit: https://github.com/Azure/azure-cli-extensions/pull/3711/commits/566b86f8fea332db6905b0bb605995485274775f

Reply:
not sure why its not showing up here

Reply:
fixed